spacepy/pybats/bats.py

```diff
@@ -1486,6 +1590,8 @@ def add_grid_plot(self, target=None, loc=111, do_label=True,
            Set subplot location.  Defaults to 111.
         do_label : boolean
            Adds resolution legend to righthand margin.  Defaults to True.
+        do_fill : boolean
+           If true, fill blocks with color indicating grid resolution.
         show_nums : boolean
```

Comment:
`bool` not `boolean`...I know this was wrong before, but since you're introducing another one here, probably time to fix it :)

Reply:
Changed.

Reply:
Fixed.